amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/Lease.java

```diff
@@ -84,6 +84,12 @@
      * @return pending checkpoint, possibly null.
      */
     private ExtendedSequenceNumber pendingCheckpoint;
+
+    /**
+     * @return pending checkpoint state, possibly null.
+     */
+    private byte[] pendingCheckpointState;
```

Comment:
I was leaning towards keeping it since we persist it in DDB which is what we've used as the common factor for the fields we include, but I don't have a strong preference on it. If you feel strongly I can remove it in next revision.